configuration_files/pose_graph.lua

```diff
@@ -63,8 +63,8 @@ POSE_GRAPH = {
   matcher_rotation_weight = 1.6e3,
   optimization_problem = {
     huber_scale = 1e1,
-    acceleration_weight = 1e3,
-    rotation_weight = 3e5,
+    acceleration_weight = 1.1e2,
+    rotation_weight = 1.6e4,
```

Comment:
These params are also in the Sphinx docs. I think you should document the new behavior there (https://github.com/cartographer-project/cartographer/blob/master/docs/source/configuration.rst).

Reply:
What about the documentation do you think is no longer accurate?

Reply:
> Scaling parameter for the IMU [...] term.

This is not wrong, but there could be a few words that there is an additional time-dependent factor affecting the final weight of the term.

Reply:
I disagree. If we wanted to describe the implementation sure, but briefly mentioning the time dependency in the parameter description would be misleading for the following reason:
This time dependency in the implementation is necessary since we do not have equal time steps because of the motion filter which made it hard/impossible to choose a good value. This change has the additional benefit of decoupling time step size from the choice of this parameter even in the constant time step case, i.e. now users should have to think less about the interaction between these weights and the interval between consecutive nodes.